_data/engine-cli/docker_push.yaml

```diff
@@ -1,23 +1,17 @@
 command: docker push
 short: Push an image or a repository to a registry
-long: |-
-  Use `docker push` to share your images to the [Docker Hub](https://hub.docker.com)
-  registry or to a self-hosted one.
-
-  Refer to the [`docker tag`](tag.md) reference for more information about valid
-  image and tag names.
-
-  Killing the `docker push` process, for example by pressing `CTRL-c` while it is
-  running in a terminal, terminates the push operation.
-
-  Registry credentials are managed by [docker login](login.md).
-
-  ### Concurrent uploads
-
-  By default the Docker daemon will push five layers of an image at a time.
-  If you are on a low bandwidth connection this may cause timeout issues and you may want to lower
-  this via the `--max-concurrent-uploads` daemon option. See the
-  [daemon documentation](dockerd.md) for more details.
+long: "Use `docker push` to share your images to the [Docker Hub](https://hub.docker.com)\nregistry
```

Comment:
@tiborvass I don't see anything in the source file that is causing this. Why is the YAML generator not putting the  `|-`?